docs/src/modules/pageDemo.js

```diff
@@ -1041,55 +1065,58 @@ class PageDemo extends React.PureComponent {
+                                    columns={[
+                                        {
+                                            accessor: 'name',
+                                            fontSize: 'medium',
+                                            fontWeight: 'semibold',
+                                            header: false,
+                                            width: '100%',
+                                        }, {
+                                            accessor: 'campus',
+                                            fontWeight: 'bold',
+                                            header: 'Campus',
+                                        }, {
+                                            accessor: (d) => moment.unix(d.createdOn).utc().format('L'),
+                                            fontWeight: 'bold',
+                                            header: 'Created On',
+                                        },
+                                    ]}
+                                    data={[
+                                        {
+                                            campus: 'Lake Forest',
+                                            createdOn: 1259668810,
+                                            id: 1,
+                                            name: 'First Time Visitor',
+                                        }, {
+                                            campus: 'Lake Forest',
+                                            createdOn: 1159668810,
+                                            id: 2,
+                                            name: 'Second Time Visitor',
+                                        }, {
+                                            campus: 'Anaheim',
+                                            createdOn: 1152668810,
+                                            id: 3,
+                                            name: 'Class 101 Invite',
+                                        },
+                                    ]}
+                                />
                             )}
 
-                            <Page.DataGroups
-                                columns={mainCoulumn}
-                                data={mainData}
+                            <div
                                 style={{
                                     backgroundColor: '#fff',
-                                    padding: '22px',
+                                    margin: '0 -22px',
                                 }}
-                            />
+                            >
+                                <Page.DataGroups
+                                    columns={mainCoulumn}
+                                    data={mainData}
+                                />
+                            </div>
                         </Page.Content>
                     </Page.Container>
                 </Page>
```

Comment:
Not blocking, but I'm sensing that the Demo Page is morphing from a generic "list of things" page to be more like the person 2.0 page.  Do we perhaps need multiple independent Demo Pages for different use cases?

Reply:
I think this is a good idea, I will try to keep it in mind when have more time to give.

Reply:
Yep, fair enough.